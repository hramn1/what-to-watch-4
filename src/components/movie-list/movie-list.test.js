import React from "react";
import renderer from "react-test-renderer";
import MovieList from "./movie-list.jsx";
const films = [
  {
    id: `1`,
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    id: `2`,
    title: `Bohemian Rhapsody`,
    poster: `img/bohemian-rhapsody.jpg`,
  },
];
const cardFilms = {
  title: `The Grand Budapest Hotel`,
  genre: `Drame`,
  year: `2014`,
};
const onMovieCardClick = () => {};

it(`MoviesList rendering`, () => {
  const tree = renderer
.create(<MovieList
  films = {films}
  cardFilms = {cardFilms}
  onMovieCardClick={onMovieCardClick}
/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
