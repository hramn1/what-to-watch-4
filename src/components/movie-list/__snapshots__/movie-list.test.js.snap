// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList rendering 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        width="280"
      >
        <source
          type="video/webm; codecs=\\"vp8, vorbis\\""
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/bohemian-rhapsody.jpg"
        width="280"
      >
        <source
          type="video/webm; codecs=\\"vp8, vorbis\\""
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
</div>
`;
